homeassistant/components/switchbot/strings.json

```diff
@@ -58,5 +58,56 @@
         }
       }
     }
+  },
+  "entity": {
+    "binary_sensor": {
+      "calibration": {
+        "name": "Calibration"
+      },
+      "motion": {
+        "name": "[%key:component::binary_sensor::entity_component::motion::name%]"
+      },
+      "door_open": {
+        "name": "[%key:component::binary_sensor::entity_component::door::name%]"
+      },
+      "door_timeout": {
+        "name": "Door timeout"
+      },
+      "light": {
+        "name": "[%key:component::binary_sensor::entity_component::light::name%]"
+      },
+      "door_unclosed_alarm": {
+        "name": "Door unclosed alarm"
+      },
+      "door_unlocked_alarm": {
+        "name": "Door unlocked alarm"
+      },
+      "door_auto_lock_paused": {
+        "name": "Door auto-lock paused"
+      }
+    },
+    "sensor": {
+      "bluetooth_signal": {
+        "name": "Bluetooth signal strength"
```

Comment:
Can we drop "strength" to keep the name shorter?

Reply:
Done and fixed tests